.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+
+jobs:
+
+  deploy_unstable:
+    docker:
+      - image: circleci/golang:1.11
+    working_directory: /go/src/github.com/replicatedcom/statsd-graphite
```

Comment:
Will it actually put the checked out code into this folder?

Reply:
ya

Reply:
which makes no sense